File: volleytrain/src/api/NamedBusinessObject.js
import BusinessObject from "./BusinessObject";

/*
Basisklasse fuer alle BO's welche einen Namen besitzen
*/

export default class NamedBusinessObject extends BusinessObject{

	constructor(){
        super();
        this.name = null;
    }
    
	/*
	Erhalte Namen
	*/
	getName(){
		return this.name;
	}
	/*
	Setze Namen
	*/
	setName(aName){
		this.name = aName;  
	}
}

File: src/server/bo/NamedBusinessObject.py
from server.bo.BusinessObject import BusinessObject


class NamedBusinessObject(BusinessObject):
    '''Base class for all business objects with a name
    every business object can features a name
    '''

    def __init__(self):
        super().__init__()
        self._name = None

    def get_name(self):
        return self._name

    def set_name(self, name):
        self._name = name
